consensus/db/rawdb.go

```diff
@@ -56,9 +56,7 @@ func (db *rawDataBase) Sync() error {
 }
 
 func (db *rawDataBase) GarbageCollect() error {
```

Comment:
Check the error raised if for not compacting. Put on utils the logic to filter the error